src/index.js

```diff
@@ -0,0 +1,37 @@
+import './style.css';
+
+const addBtn = document.querySelector('#add-btn');
+const todoList = document.querySelector('.todo-list');
+const inputField = document.querySelector('#input-field');
+const clearAll = document.querySelector('.clearAll');
+
+const todoListArray = [];
+
+const addnew = () => {
+  todoListArray.push(inputField.value);
+  const div = document.createElement('div');
+  div.classList.add('div-active');
+  div.innerHTML = inputField.value;
+  todoList.insertBefore(div, clearAll);
+  inputField.value = '';
+  const checkbox = document.createElement('input');
+  checkbox.type = 'checkbox';
+  div.appendChild(checkbox);
+  checkbox.classList.add('checkbox');
+  checkbox.addEventListener('click', (e) => {
```

Comment:
- [ ] Sometimes when I click on the checkbox, the line-through doesn't go away, please fix this behavior.
![Screen Recording 2021-12-09 at 6 22 49 PM](https://user-images.githubusercontent.com/47994600/145454749-5d679f66-75cc-4380-aee8-603712003475.gif)